pkg/plugins/diff/service.go

```diff
@@ -104,15 +105,20 @@ type Service struct {
 func (s *Service) RunDiff(ctx context.Context, diffType string, diffParams Params) (Response, error) {
 	d, closeClient, err := s.pluginHandler.LoadPluginClient(diffType)
 	if err != nil {
-		return Response{}, err
+		logging.Default().WithError(err).Errorf("encountered err while loading the plugin client")
```

Comment:
Error logs should include a context in order to be useful. At the very least it will provide the request ID and allow threading the error to other lakeFS operations on that request.  Also they can include useful parameters, e.g.

```suggestion
		logging.FromContext(ctx).WithError(err).WithField("diff_type", diffType).WithField("diff_params", fmt.Sprintf("%+v", diffParams).Error("Failed to load plugin client")
```

Also below.

Reply:
done